pom.xml

```diff
@@ -63,6 +63,36 @@
       <artifactId>access-modifier-suppressions</artifactId>
       <version>1.16</version>
     </dependency>
+    <dependency>
+      <groupId>net.java.dev.jna</groupId>
+      <artifactId>jna</artifactId>
+      <version>5.4.0</version>
+    </dependency>
+    <dependency>
+      <groupId>net.java.dev.jna</groupId>
+      <artifactId>jna-platform</artifactId>
+      <version>5.4.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jcl-over-slf4j</artifactId>
+      <version>2.0.0-alpha0</version>
```

Comment:
It's better to specify the version as a new property in the `<properties>` element, and reference it here.